service-deployment/bootstrap/webportal/webportal-ds.yaml.template

```diff
@@ -35,10 +35,12 @@ spec:
         image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}webportal
         imagePullPolicy: Always
         env:
-        - name: REST_SERVER_URI
-          value: {{ clusterinfo['webportalinfo']['rest_server_uri'] }}
-        - name: CLUSTER_STATUS_URI
-          value: {{ clusterinfo['webportalinfo']['cluster_status_uri'] }}
+        - name: REST_SERVER_ROOT_URL
+          value: {{ clusterinfo['webportalinfo']['rest_server_root_url'] }}
```

Comment:
we have used the `XXX_URI`, so please be consistent if there's no problem with it.
why should we need to change the variable name? you have to change all similar variable names in webportal and rest server, and that may lead to new unexpected bugs